lib/Target/LLVMIR/ModuleTranslation.cpp

```diff
@@ -84,6 +84,8 @@ llvm::Constant *ModuleTranslation::getLLVMConstant(llvm::Type *llvmType,
     return llvm::ConstantFP::get(llvmType, floatAttr.getValue());
   if (auto funcAttr = attr.dyn_cast<SymbolRefAttr>())
     return functionMapping.lookup(funcAttr.getValue());
+  if (auto boolAttr = attr.dyn_cast<BoolAttr>())
```

Comment:
Can you add a test case to exercise this? (to test/Target/llvmir.mlir)

Reply:
It appears that the problem we had may have been resolved another way.  We're still checking.